nautobot/extras/jobs.py

```diff
@@ -278,31 +441,10 @@ def _get_file_vars(cls):
 
     @property
     def job_result(self):
+        if self._job_result is None:
+            self._job_result = self.get_job_result()
```

Comment:
Make this a `@cached_property` and then you can just `return self.get_job_result()` here without having to explicitly save it on the instance.

Reply:
Right, so just replace it with the getter and throw a party.